documentation.md

```diff
@@ -1,34 +1,35 @@
 - ## Getting Started
     - [Installation](/docs/installation)
-	- [Writing Tests](/docs/writing-tests)
-	- [Underlying Test Case](/docs/underlying-test-case)
-	- [Assertions](/docs/assertions)
-	- [Expectations](/docs/expectations)
-	- [Setup And Teardown](/docs/setup-and-teardown)
-	- [Higher Order Tests](/docs/higher-order-tests)
-	- [Custom Helpers](/docs/helpers)
-	- [IDE Plugins](/docs/ide-plugins)
-	- [Exceptions & Errors](/docs/exceptions-and-errors)
-	- [Groups Of Tests](/docs/groups)
-	- [Skipping Tests](/docs/skipping-tests)
-	- [Datasets](/docs/datasets)
-	- [Coverage](/docs/coverage)
-	- [CLI Options](/docs/cli-options)
-- ## Plugins
-	- [Laravel](/docs/plugins/laravel)
-	- [Livewire](/docs/plugins/livewire)
-	- [Mock](/docs/plugins/mock)
-  	- [Money](/docs/plugins/money)
-	- [Mutation Testing](/docs/plugins/mutation-testing)
-	- [Faker](/docs/plugins/faker)
-	- [Global Assertions](/docs/plugins/global-assertions)
-	- [Snapshots](/docs/plugins/snapshots)
-	- [Time](/docs/plugins/time)
-	- [Watch](/docs/plugins/watch)
-	- [Creating Plugins](/docs/plugins/creating-plugins)
+    - [Writing Tests](/docs/writing-tests)
+    - [Underlying Test Case](/docs/underlying-test-case)
+    - [Assertions](/docs/assertions)
+    - [Expectations](/docs/expectations)
+    - [Setup And Teardown](/docs/setup-and-teardown)
+    - [Higher Order Tests](/docs/higher-order-tests)
+    - [Custom Helpers](/docs/helpers)
+    - [IDE Plugins](/docs/ide-plugins)
+    - [Exceptions & Errors](/docs/exceptions-and-errors)
+    - [Groups Of Tests](/docs/groups)
+    - [Skipping Tests](/docs/skipping-tests)
+    - [Datasets](/docs/datasets)
+    - [Coverage](/docs/coverage)
+    - [CLI Options](/docs/cli-options)
+- ## Plugins 
+    - [Faker](/docs/plugins/faker)
```

Comment:
It's a nice touch to reorder the plugin in alphabetical order!
However, the internal linking will be broken unless every concerned doc page is edited to reflect the new order.

Reply:
Reverted the menu 👍

Reply:
We can update that in a separate PR